### DISCUSSION TITLE
Multiple CRUD Apps for the same model

### DISCUSSION THREAD

**Discussion by blackfyre** [TiagoSilvaPereira/vemto-issues#366] [Ideas & Suggestions]
**Describe the bug**
The app allows multiple CRUD apps to be created for the same model, but after generating the app, while the multiple apps are accessible through the menu, the actual view is the last one generated for the model.
This is not desirable as in some use cases, it's necessary to create different CRUD apps for the same model.

**To Reproduce**
Create 2 CRUD apps for the same model with different fields added to the apps.

**Expected behavior**
The 2 CRUD apps should work independently of each other, much like how Laravel Nova's resources work in this regard.

**Comment by henriquek3** [comment]
Eu também aguardo essa solução pois no meu contexto eu preciso de um modelo para Pessoas que contenham os CRUD de Clientes e Fornecedores, estes possuem atributos e objetos master-details diferentes entre si.

**Comment by TiagoSilvaPereira** [comment]
@blackfyre thank you for reporting, I'll fix it on future versions

**Comment by blackfyre** [comment]
> @blackfyre thank you for reporting, I'll fix it on future versions

If I can help with it, let me know 😁

**Comment by TiagoSilvaPereira** [comment]
I'll add this as suggestion for Vemto v2 (which will bring a better architecture to support this type of feature)

**Reply by blackfyre** [reply]
I really look forward to Vemto2 😄
